public/index.html

```diff
@@ -9,8 +9,6 @@
         <meta name="author" content="https://github.com/blumilksoftware/website/contributors">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-        <title>Blumilk Software Development</title>
```

Comment:
Maybe we should leave this `title` tag here for some default behaviour?

Reply:
What about that?

Reply:
We add title tag with this line of code `<Meta title={ $_('title.blumilk-software-development') }/>`

Reply:
Okaaay, but it's javascript induced and bots aren't using JS probably. I would not like to have empty header in Google after this change.